tests/behat/features/capabilities/account/edit-profile-page.feature

```diff
@@ -0,0 +1,12 @@
+@api
```

Comment:
I believe this page could have been name edit-profile-redirection.
What do you think?

Reply:
I follow the pattern used in https://github.com/goalgorilla/open_social/pull/4097
So if needs be changed here, I guess should be better change there as well.

Reply:
I have to admit i didnt look at the file name there - I guess yes you can update the file name part of this PR